usecases/config/environment.go

```diff
@@ -462,9 +462,13 @@ const (
 
 const VectorizerModuleNone = "none"
 
-// DefaultGossipBindPort uses the hashicorp/memberlist default
-// port value assigned with the use of DefaultLocalConfig
-const DefaultGossipBindPort = 7946
+// DefaultGossipBindPort and DefaultGossipAdvertisePort uses the
+// hashicorp/memberlist default port value assigned with the use of
+// DefaultLocalConfig
+const (
+	DefaultGossipBindPort = 7946
+	DefaultAdvertisePort  = 7946
```

Comment:
can we not have a default value for advertise port? to only set it explicitly is someone provides a value for this setting? my reasoning behind this is that we would introduce this setting in an optional manner so that it would not add any other settings to memberlist by default, only if someone wants to set it